modules/passkey/package.json

```
@@ -48,25 +48,25 @@
   "devDependencies": {
     "@noble/curves": "^1.4.0",
     "@nomicfoundation/hardhat-ethers": "^3.0.6",
-    "@nomicfoundation/hardhat-network-helpers": "^1.0.10",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.11",
     "@nomicfoundation/hardhat-toolbox": "^5.0.0",
     "@safe-global/mock-contract": "^4.1.0",
     "@safe-global/safe-4337": "workspace:^0.3.0-1",
     "@safe-global/safe-4337-local-bundler": "workspace:^0.0.0",
+    "@safe-global/safe-contracts": "^1.4.1-build.0",
     "@simplewebauthn/server": "^10.0.0",
-    "@types/node": "^20.14.0",
+    "@types/node": "^20.14.8",
     "dotenv": "^16.4.5",
-    "ethers": "^6.12.1",
-    "hardhat": "^2.22.3",
+    "ethers": "^6.13.1",
+    "hardhat": "^2.22.5",
     "hardhat-deploy": "^0.12.4",
     "solhint": "^5.0.1",
     "ts-node": "^10.9.2",
-    "typescript": "^5.4.5"
+    "typescript": "^5.5.2"
   },
   "dependencies": {
-    "@account-abstraction/contracts": "^0.7.0",
-    "@openzeppelin/contracts": "^5.0.0",
-    "@safe-global/safe-contracts": "^1.4.1-build.0",
+    "@account-abstraction/contracts": "0.7.0",
+    "@openzeppelin/contracts": "^5.0.2",
```

This is a transitive dependency from account-abstraction (whose package has a bug and doesn't correctly include the contracts as a dependency) and uses version 5.0.0:

https://github.com/eth-infinitism/account-abstraction/blob/7af70c8993a6f42973f520ae0752386a5032abe7/yarn.lock#L901-L904

I would suggest marking this as:

```suggestion
    "@openzeppelin/contracts": "=5.0.0",
```

To indicate this is an exact version on purpose.

---

Actually, they do include it as a dependency (I created a PR to fix it). We have it because hardhat can't resolve pnpm modules correctly: https://github.com/NomicFoundation/hardhat/issues/4292

---

Ah, good find.

Should we use 5.0.0 though, so it matches the version in the repository?

---

We only use the AA dependency in tests and do not depend on the same bytecode, so I think it's fine. Openzeppelin package is also following semver so I think it's in the allowed version range.

---

Wait - if its only used in tests, then I think it should be a `devDependency`.